ui/i18n/src/main/res/values/strings.xml

```diff
@@ -610,7 +610,7 @@
     </plurals>
     <string name="auto_enable_label">Automatically activate the sleep timer when pressing play</string>
     <string name="auto_enable_label_with_times">Automatically activate the sleep timer when pressing play between %s and %s</string>
-    <string name="auto_enable_change_times">Change times</string>
+    <string name="auto_enable_change_times">Change time range</string>
```

Comment:
```suggestion
    <string name="auto_enable_change_times">Change time slot</string>
```
https://dictionary.cambridge.org/dictionary/english/time-slot

https://dictionary.cambridge.org/dictionary/english/date-range

Reply:
Isn't this exactly what a range does? "a number of dates that includes a particular start and finish date and all dates in between". You can select a start time and end time, and for all times in between, the sleep timer is auto-enabled. Slot would mean that there is a discrete selection of times for which something can happen (eg the timer stops at the next full 15 minutes).

Reply:
'Time range' just sounds weird to me.

A time slot is "a time when something … is planned to happen", in our case: the automatic activation of the sleep timer. So I think that works.

We can check with native speakers.

Reply:
I guess I could pitch in as a native English speaker: from looking at the sleep timer dialog and seeing the context for this string, "time range" makes a lot more sense. You're selecting a range of time between two points on the clock dialog. "Slot" isn't really intuitive here.

Reply:
Thanks for pitching in @TacoTheDank!